src/Heatmap/CalendarHeatmap.tsx

```diff
@@ -47,11 +47,15 @@ const xAxisLabelFormat = (start: Date) => (weeks: number) =>
 
 export const CalendarHeatmap: FC<Partial<CalendarHeatmapProps>> = ({
   view,
-  series,
```

Comment:
`series` is not consumed in the component, by removing this from destructure, it will be pass through with `rest` into the Heatmap component